index.html

```diff
@@ -1790,15 +1790,25 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="caption">
 			<rdef>caption</rdef>
 			<div class="role-description">
-				<p>Visible content that names, and may also describe, a <rref>figure</rref>, <rref>table</rref>, <rref>grid</rref>, or <rref>treegrid</rref>.</p>
+				<p>Visible content that names, or describes a <rref>group</rref>, <rref>figure</rref>, <rref>table</rref>, <rref>grid</rref>, <rref>radiogroup</rref>, or <rref>treegrid</rref>.</p>
 				<p>When using <code>caption</code> authors SHOULD ensure:</p>
```

Comment:
Based on your comment, maybe rather than a "SHOULD", this should be a general recommendation with the justification that is what would match the HTML pattern?

On the other hand, for table/grid, it might actually be a SHOULD. I haven't tested what happens if you throw a random caption in the middle of rows 😆

Reply:
Right?  I'm not convinced this needs to be a 'should'.... but if it remains it needs to be clarified that if it is a child of these elements, it needs to reflect the way it's used in HTML...

Reply:
after thinking more about this, i'm leaving this as is for now.  we could always change in another PR if need be.